docs/FRSKY_R9_TX.md

```diff
@@ -0,0 +1,171 @@
+
+#### Flash the ELRS Bootloader ####
+
+1. Download the [r9m\_elrs\_bl.frk](https://github.com/ExpressLRS/ExpressLRS/blob/master/src/bootloader/r9m_elrs_bl.frk?raw=true) file from the ExpressLRS git repository and copy it to the FIRMWARE folder on your radio's SD card.
+2. Install the R9M module in your radio and power it on.
+3. Enter the System Menu and navigate using the page buttons to the SD card page.
+4. Scroll to the FIRMWARE folder and select the r9m\_elrs\_bl.frk file. Choose "Flash external module".
+5. Follow the instructions in the next section to flash the mLRS firmware.
+If you have previously flashed via ST-Link, you can install the ELRS bootloader using ST-Link instead:
+
+1. Download the [r9m\_bootloader.bin](https://github.com/ExpressLRS/ExpressLRS/blob/master/src/bootloader/r9m_bootloader.bin?raw=true) file from the ExpressLRS git repository.
+2. Connect to the R9M and perform a full chip erase.
+3. Flash the bootloader.bin to the beginning of the flash (address 0x8000000).
+4. Follow the instructions in the next section to flash the mLRS firmware.
+
+#### Flash/Update the mLRS Firmware ####
+
+Once the ELRS bootloader is installed, the following precedure can be followed to install/update the mLRS firmware.
+
+1. Download the latest mLRS firmware, or build with MLRS\_FEATURE\_ELRS\_BOOTLOADER defined.
+2. Copy the latest tx-R9M-f103c8-elrs-bl-v*.elrs file to FIRMWARE folder on your radio's SD card. Note the "elrs-bl" in the file name.
+   - Note: On bw radios it will be necessary to rename the firmware file to a shorter file name such as "tx-r9m.elrs"
+3. Enter the System Menu and navigate using the page buttons to the SD card page.
+4. Select the FIRMWARE folder and scroll to and select the tx-R9M-f103c8-elrs-bl-v*.elrs file. Choose "Flash external ELRS".
+
+#### Install the mLRS Lua Script ####
+
+Once mLRS is installed, you can verify that everything is working as expected using the Lua script, instructions are found [here](LUA.md).
+
+### Flash/Update via ST-Link ###
+
+Notes:
+  - Flashing any Frsky R9 module with ST-Link means that it is not possible to revert back to the original Frsky firmware. However, it is possible to switch to ExpressLRS.
+  - In the ExpressLRS docs it is suggested to download and use the "ST-LINK Utility" software. This software is pretty outdated (NRND = not recommended for new designs). Please use the new and recommended tool "STM32CubeProgrammer".
+
+1. Download the non-"elrs-bl" firmware, which can be identified by the ".hex" extension and that the label "elrs-bl" is not contained in the firmware file name. 
+2. Follow the instructions [here](https://www.expresslrs.org/quick-start/transmitters/frsky-r9modules/#via-stlink).
+
+#### Install the mLRS Lua Script ####
```

Comment:
a bit strange here
should it be ## xx ## titled?

Reply:
Updated to ##